Dockerfile

```diff
@@ -0,0 +1,18 @@
+# base image
+FROM node:12.2.0-alpine
+   
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install and cache app dependencies
+COPY package.json /app/package.json
+
+RUN npm install
+
+EXPOSE 3000
+
+# start app
+CMD ["npm", "start"]
```

Comment:
We use yarn as our [default](https://github.com/SoftboxLab/juggernaut#patterns) package manager, so I will ask you to change `npm`  to `yarn`.